### PR TITLE
Modal: Reverse the button order when marked as dangerous

### PR DESCRIPTION
**Pull request by thgreasi**
image

image

Resolves: #1210
Change-type: minor
See: https://www.flowdock.com/app/rulemotion/r-product/threads/Q5_H8u9NO475mHQcE2BNOgXKUT0
Signed-off-by: Thodoris Greasidis <thodoris@balena.io>

---
##### Contributor checklist

- [ ] I have rebuilt the README with `npm run build:docs`
- [ ] I have regenerated screenshots for any affected components with `npm run generate-screenshots`
- [ ] I have regenerated jest snapshots for any affected components with `npm run jest -- -u`

##### Reviewer Guidelines
- When submitting a review, please pick:
  - '*Approve*' if this change would be acceptable in the codebase (even if there are minor or cosmetic tweaks that could be improved).
  - '*Request Changes*' if this change would not be acceptable in our codebase (e.g. bugs, changes that will make development harder in future, security/performance issues, etc).
  - '*Comment*' if you don't feel you have enough information to decide either way (e.g. if you have major questions, or you don't understand the context of the change sufficiently to fully review yourself, but want to make a comment)
---

### CODE REVIEW

**Comment by thgreasi** [comment]
@balena-ci rebase

File: src/components/Modal/index.tsx [hunk 1]
```diff
@@ -100,21 +101,31 @@ class Modal extends React.Component<ThemedModalProps, any> {
 		const { width, theme, ...props } = this.props;
 
 		const cancelButtonProps = Object.assign(
-			{ style: { marginRight: 20 } },
+			{ ml: 20 },
```
Comment (sradevski) [inline src/components/Modal/index.tsx:104]: Let's set it to `3` instead for consistency

Comment (sradevski) [reply src/components/Modal/index.tsx:104]: @thgreasi if you don't wanna rerun everything, feel free to self-certify as it's not a big deal

Comment (thgreasi) [reply src/components/Modal/index.tsx:104]: @sradevski does self-certifying allow you to bypass screenshot testing?

Comment (sradevski) [reply src/components/Modal/index.tsx:104]: no :)

Comment (thgreasi) [reply src/components/Modal/index.tsx:104]: Oh ok I now got what you mean :)

Comment (thgreasi) [reply src/components/Modal/index.tsx:104]: Actually let me open a new PR on top of this.
See: https://github.com/balena-io-modules/rendition/pull/1214

**Review by sradevski — APPROVED** [review]

**Pull request merged**